### PR TITLE
Add common checks and dependencies on this repository

### PR DESCRIPTION
**Pull request by marcocarvalho**
**Description**: Add some common checks and dependencies on this repository.
- [x] codeowners
- [x] W  I   P
- [x] pull request template
- [ ] ~CircleCI~
- [x] dependabot (https://github.com/CareMessagePlatform/check_if_merged/pull/4)

**JIRA**: NO-JIRA

**How to Verify**: Code review and see automate test.

**Notes**
I think there is no need for CircleCI build. We can automate the test with GH workflows.
**Pros:** It is in the exactly environment where the action will run allowing us focus on the GH Action script test only.
**Cons:** It's a shell script. It will be hard to maintain if the test need to grow and became complex.

image

**SIMPLE PR TASKS**
- [x] Pick at least one reviewer
- [x] Merge to *Staging*
- [x] Add necessary tags(CI, Code Review)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+**Notes**: Additional notes that should be taken in account. e.g. *All projects must have the pull request template.*
+
+DELETE ONE OF THESE TASK LISTS
+**SIMPLE PR TASKS**
+- [ ] Pick at least one reviewer
+- [ ] Merge to *Staging*
+- [ ] Add necessary tags(QA, CI, Code Review, Code analysis, Code Coverage)
+- [ ] Move JIRA story into Ready-for-QA
+
+**QA TASKS**
```
Comment (danielnc) [inline .github/PULL_REQUEST_TEMPLATE.md]: we dont have api tasks

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] Verify API documentation
+- [ ] Verify API endpoints
+
+**DEPLOYMENT TASKS**
```
Comment (danielnc) [inline .github/PULL_REQUEST_TEMPLATE.md]: no need for deployment or migration

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] Request Data to support the deployment.
+- [ ] Evidences of successful migration on Staging (Logs and Numbers, enqueued jobs).
+
+**INTERMEDIARY PR TASKS**
```
Comment (danielnc) [inline .github/PULL_REQUEST_TEMPLATE.md]: there wont be intermediary prs for this repo

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+**JIRA**: [CM-111](http://jira.caremessage.org/browse/CM-111)
+
+**How to Verify**: Describe how the changes can be verified. e.g. *Check if the template has all important information that needs to be described in a pull request.*
+
+**Notes**: Additional notes that should be taken in account. e.g. *All projects must have the pull request template.*
+
+DELETE ONE OF THESE TASK LISTS
+**SIMPLE PR TASKS**
+- [ ] Pick at least one reviewer
+- [x] Merge to *Staging*: See [test, It trys to merge to staging and see if the action works](https://github.com/CareMessagePlatform/check_if_merged/blob/master/.github/workflows/check_if_merged.yml)
```
Comment (marcocarvalho) [inline .github/PULL_REQUEST_TEMPLATE.md]: This link will be true when this PR is merged.

For now, it this file: https://github.com/CareMessagePlatform/check_if_merged/pull/3/files#diff-195a635ad245ded9076330e31134bd80R12

**Comment by marcocarvalho** [comment]
@danielnc see **notes** of this PR about CircleCI. If you agree to just use GH workflows for now, Can you remove CircleCI integration on this project? I don't have  the permission on CI to do that.

**Pull request merged**